internal/dinosaur/pkg/cmd/fleetmanagerclient/client.go

```diff
@@ -13,16 +13,59 @@ import (
 
 var (
 	singletonInstance sync.Once
-	client            *fleetmanager.Client
+	ocmClient         *fleetmanager.Client
+
+	ocmClientStaticToken         *fleetmanager.Client
+	singletonStaticTokenInstance sync.Once
 )
 
 const (
 	defaultFleetManagerEndpoint = "http://localhost:8000"
 	fleetManagerEndpointEnvVar  = "FMCLI_FLEET_MANAGER_ENDPOINT"
 	ocmRefreshTokenEnvVar       = "OCM_TOKEN"
+	staticTokenEnvVar           = "STATIC_TOKEN"
 )
 
-// AuthenticatedClientWithOCM returns a rest client to the fleet-manager and receives the OCM refresh token.
+// AuthenticatedClientWithStaticToken returns a rest ocmClient to the fleet-manager using a static token.
```

Comment:
```suggestion
// AuthenticatedClientWithStaticToken returns a rest client for fleet-manager API using a static OCM token for authentication.
```

I'm still struggling with calling this an ocmClient. In my opinion OCM is an API/CLI tool we depend on. If something is called ocmClient, I expect it to have the sole purpose of helping me communicate with the OCM API. But this is returning a client for fleet-manager API with a preconfigured `STATIC_TOKEN` that should be set to a valid static token for ocm.